pkg/util/limiter/utilization_test.go

```diff
@@ -17,7 +18,7 @@ func TestUtilizationBasedLimiter(t *testing.T) {
 
 	setup := func(t *testing.T, cpuLimit float64, memoryLimit uint64) (*UtilizationBasedLimiter, *fakeUtilizationScanner) {
 		fakeScanner := &fakeUtilizationScanner{}
-		lim := NewUtilizationBasedLimiter(cpuLimit, memoryLimit, log.NewNopLogger())
+		lim := NewUtilizationBasedLimiter(cpuLimit, memoryLimit, log.NewNopLogger(), prometheus.NewPedanticRegistry())
```

Comment:
WDYT about asserting on the exported metrics below, at the end of each test case? We could assert on the CPU metric in test cases testing the CPU, and on memory metric in test cases testing the memory.

Reply:
@pracucci that is probably a good idea!

Reply:
Done.